chat-commands.js

```diff
@@ -133,8 +133,8 @@ exports.commands = {
 		if (target.includes(separator)) {
 			const params = target.split(separator);
 			let output = [];
-			for (let i = 0; i < params.length; i++) {
-				output.push(Chat.escapeHTML(params[i]));
+			for (const param of params) {
+				output.push(Chat.escapeHTML(param));
 			}
```

Comment:
``const output = params.map(Chat.escapeHTML)`` ?

Reply:
Ehh, I was just trying to refactor existing loops. `Array.prototype.map` is slower than a for loop anyway...